plus/tool/src/main/webapp/WEB-INF/templates/form.html

```diff
@@ -182,6 +182,21 @@ <h1><span th="#{plus.tool.tenant.title}">Tenant:<span>
     let inp = document.getElementById(id+'-boolean');
     inp.value = checked ? 'true' : 'false';
 }
+</script>
+
+<script th:inline="javascript">
+    function validateTenantForm() {
+        let issuerInput = document.getElementById("issuer");
+        let issuerValue = issuerInput.value.trim();
+        let issuerInputErrorMessage = /*[[#{plus.tenant.form.issuer.error.message}]]*/;
+        if (issuerValue.endsWith("/")) {
+            alert(issuerInputErrorMessage);
+            issuerInput.focus();
+            issuerInput.scrollIntoView();
```

Comment:
Do you need to use `scrollIntoView()` here? I think `focus` should do the job. Also, keep in mind that the combination of both `focus()` and `scrollIntoView()` can affect accessibility. I will try to avoid it, If I can. Why do you have to use focus here?

Reply:
Thank you for the suggestion! I used ```scrollIntoView()``` because when I searched how to use ```focus()``` I found some people used it. And I am glad to know that ```focus()``` can do the job, so I am gonna remove ```scrollIntoView()```.